### PR TITLE
fix(site): anchor offset

### PR DESCRIPTION
**Pull request by GaoNeng-wWw**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #1448 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by gimmyhehe — DISMISSED** [review]

File: examples/sites/src/views/components/components.vue [hunk 1]
```diff
@@ -443,7 +444,7 @@ export default defineComponent({
 })
 </script>
 
-<style lang="less">
+<style lang="less" scoped>
```
Comment (gimmyhehe) [inline examples/sites/src/views/components/components.vue:444]: 感谢您的贡献~  之前这里没有加scoped是我们遗留的问题，但这里加了scoped后，锚点的字体会变大，因为.tiny-anchor-link的样式没有了，需要加deep。

File: examples/sites/src/views/components/components.vue [hunk 2]
```diff
@@ -128,6 +128,7 @@
           :key="anchorRefreshKey"
           mask-class="custom-active-anchor"
           @link-click="handleAnchorClick"
+          class="anchor"
```
Comment (gimmyhehe) [inline examples/sites/src/views/components/components.vue:131]: 这个新增的类名是没有用到么~

File: examples/sites/src/views/components/components.vue (hunk 1 above)
Comment (gimmyhehe) [reply examples/sites/src/views/components/components.vue:444]: PR#1473已新增deep

File: examples/sites/src/views/components/components.vue (hunk 2 above)
Comment (GaoNeng-wWw) [reply examples/sites/src/views/components/components.vue:131]: 好像是，我改一下

**Pull request closed without merging**